Multisig/UI/Onboarding/LaunchView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  LaunchView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 27.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+extension HorizontalAlignment {
+    private enum CenterHorizontalAlignment : AlignmentID {
+        static func defaultValue(in d: ViewDimensions) -> CGFloat {
+            return d[HorizontalAlignment.center]
+        }
+    }
+
+    static let centerHorizontalAlignment = HorizontalAlignment(CenterHorizontalAlignment.self)
+}
+
+extension VerticalAlignment {
+    private enum CenterVerticalAlignment : AlignmentID {
+        static func defaultValue(in d: ViewDimensions) -> CGFloat {
+            return d[VerticalAlignment.center]
+        }
+    }
+
+    static let centerVerticalAlignment = VerticalAlignment(CenterVerticalAlignment.self)
+}
+
+extension Alignment {
+    static let centerAlignment = Alignment(horizontal: .centerHorizontalAlignment, vertical: .centerVerticalAlignment)
+}
+
+struct LaunchView: View {
+    @Binding var acceptedTerms: Bool
+    @State var isAgreeWithTermsPresented = false
+
+    var body: some View {
+        GeometryReader { gp in
+            ZStack(alignment: .centerAlignment) {
+                // anchor to position text image in the center of the screen
+                Rectangle()
+                    .frame(width: 0, height: 0)
```

Comment:
Maybe the `EmptyView()` is what you need here instead of empty rectangle?  Or even make it an "CenterAnchorView" that accepts geometry proxy... don't know if that will work

Reply:
With EmptyView it does not work here and in the place where I need it for additional spacing.